### PR TITLE
Feature/add versioning

### PR DESCRIPTION
**Pull request by mikemcd3912**
*Issue #, if available:*
#125 

*Description of changes:*
Added label to the README for new labeling requirements
Updated partner Namespace files with the current versions based on available Helm chart versions
Updated Validated Partner files for each environment with the Version field and associated values

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mikemcd3912** [comment]
GH Action failed because I updated the Validated Partners files outside of the expected workflow, and is expected behavior

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@mikemcd3912  Nice work, couple of comments. We should use Automation to update the Partner list with Versions. We should not update manually. And Reason why manifestJob was removed.

File: Validated_Partners/baremetal_Partners.txt [hunk 1]
```diff
@@ -3,18 +3,18 @@ Date of Conformance Test : 2023-11-07
...
-kubecost         cost-analyzer
-nirmata          enterprise-kyverno
-lacework         polygraph
-suse             neuvector
-newrelic         newrelic-bundle
-perfectscale     perfectscale
-pulumi           pulumi-kubernetes-operator
-sysdig           sysdig-agent
-hashicorp        vault
+VENDOR_PRODUCT   VENDOR_PRODUCT_TYPE            VENDOR_PRODUCT_VERSION
```
[the remark this answers is not in the document]
Comment (mikemcd3912) [reply Validated_Partners/baremetal_Partners.txt]: Reverted to prior state, To be updated by workflow after update

File: Validated_Partners/eks-local-clusters-outposts_Partners.txt [hunk 2]
```diff
@@ -3,17 +3,17 @@ Date of Conformance Test : 2023-11-07
...
-kubecost         cost-analyzer
-nirmata          enterprise-kyverno
-lacework         polygraph
-suse             neuvector
-perfectscale     perfectscale
-pulumi           pulumi-kubernetes-operator
-sysdig           sysdig-agent
-hashicorp        vault
+VENDOR_PRODUCT   VENDOR_PRODUCT_TYPE            VENDOR_PRODUCT_VERSION
+dynatrace        dynatrace                      0.10.1
```
[the remark this answers is not in the document]
Comment (mikemcd3912) [reply Validated_Partners/eks-local-clusters-outposts_Partners.txt]: Reverted to prior state, To be updated by workflow after update

File: Validated_Partners/vsphere_Partners.txt [hunk 3]
```diff
@@ -3,18 +3,18 @@ Date of Conformance Test : 2023-11-07
...
-perfectscale     perfectscale
-pulumi           pulumi-kubernetes-operator
-sysdig           sysdig-agent
-hashicorp        vault
+VENDOR_PRODUCT   VENDOR_PRODUCT_TYPE            VENDOR_PRODUCT_VERSION
+dynatrace        dynatrace                      0.10.1
+solo.io          solo-istiod                    1.18.3-eks-a
+komodor          k8s-watcher                    1.15.5
+kong             kong-enterprise                2.27.0
+accuknox         kubearmor                      v0.10.2
```
[the remark this answers is not in the document]
Comment (mikemcd3912) [reply Validated_Partners/vsphere_Partners.txt]: Reverted to prior state, To be updated by workflow after update

File: Validated_Partners/snow_Partners.txt [hunk 4]
```diff
@@ -3,18 +3,18 @@ Date of Conformance Test : 2023-11-10
...
-perfectscale     perfectscale
-pulumi           pulumi-kubernetes-operator
-sysdig           sysdig-agent
-hashicorp        vault
+VENDOR_PRODUCT   VENDOR_PRODUCT_TYPE            VENDOR_PRODUCT_VERSION
+dynatrace        dynatrace                      0.10.1
+solo.io          solo-istiod                    1.18.3-eks-a
+komodor          k8s-watcher                    1.15.5
+kong             kong-enterprise                2.27.0
+accuknox         kubearmor                      v0.10.2
```
[the remark this answers is not in the document]
Comment (mikemcd3912) [reply Validated_Partners/snow_Partners.txt]: Reverted to prior state, To be updated by workflow after update

**Comment by mikemcd3912** [comment]
I pulled ManifestJob because it seemed like it was associated with testing and therefore would serve as a distraction from our current examples for ChronJobs in the Testers folder. Updated PR to remove that deletion only modify Version update files

**Comment by elamaran11** [comment]
> I pulled ManifestJob because it seemed like it was associated with testing and therefore would serve as a distraction from our current examples for ChronJobs in the Testers folder. Updated PR to remove that deletion only modify Version update files

It is a sample. I think our `ReadMe` references this if im not mistaken. Lets keep it as a source of sample to share to partners who ask. This helps us being not biased to share any particular partner sample to anyone!

**Review by elamaran11 — APPROVED** [review]
LG

**Comment by elamaran11** [comment]
Looks like your deployment `dynatrace-dynatrace-operator` is failing.
Specifically, it looks like these pods are failing: 
* Pod: `dynatrace-dynatrace-operator-6859dbbd89-7n2sx`.
----

**Pull request merged**